config/config.defaults.yml

```diff
@@ -238,6 +238,11 @@ default: &default_settings
   # This is completely optional and probably not needed in most cases.
   uservoice_widget_url:
 
+  # Charm connection
+  # The connection details hosted Sharetribe uses to connect to subscription management system Charm.
+  charm_secret: 
+  charm_url_base:
```

Comment:
There are specific to the hosted service, I wonder how we should control these and enable Charm only there